rpmvenv/extensions/python/venv.py

```diff
@@ -132,7 +133,9 @@ def generate(config, spec):
             ' --destination=/%{venv_install_dir}',
             '# Strip native modules as they contain buildroot paths in their'
             ' debug information',
-            'find %{venv_dir}/lib -type f -name "*.so" | xargs -r strip'
+            'find %{venv_dir}/lib -type f -name "*.so" | xargs -r strip',
+            '# Remove symlink directories (lib64 -> lib)',
+            'for link in `find %{venv_dir} -type l` ; do source=`readlink $link` ; unlink $link ; cp -r $source $link ; done'
```

Comment:
Seems like this line should only be added if the symlink support is enabled. I'd like to preserve the default behaviour as much as possible and only opt-in to new features when needed.